### DISCUSSION TITLE
Website Hosting Security Error

### DISCUSSION THREAD

**Discussion by OGEmanuel** [vercel/community#6927] [Help]
### Summary

I hosted a website here and I'm using a GoDaddy domain name. However, whenever I visit the site, I get a security error saying the site is a dangerous site

### Steps to Reproduce

visit the page https://www.switchonline.xyz/

**Comment by sahilatahar** [comment]
When you visit your Vercel-hosted site using a GoDaddy domain and see a security error, it often means there's an issue with SSL/TLS certificates or DNS settings. Here's how to fix it:

### Steps to Fix Security Error

1. **Verify DNS Settings in GoDaddy**:
   - Add an `A` record pointing to Vercel's IP (e.g., `76.76.21.21`).
   - Add a `CNAME` record pointing to `cname.vercel-dns.com`.

2. **Add Domain to Vercel**:
   - Go to your Vercel project settings.
   - Add your custom domain under the "Domains" section.

3. **Ensure SSL is Active**:
   - Vercel should automatically provision SSL certificates. Verify this in the Vercel dashboard under "Domains".

4. **Wait for DNS Propagation**:
   - DNS changes can take time to propagate. Use [What's My DNS](https://www.whatsmydns.net/) to check.

5. **Clear Browser Cache**:
   - Clear your browser cache or try accessing the site in incognito mode.
